boards/arm/adafruit_feather_m0_lora/adafruit_feather_m0_lora_defconfig

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_SAMD21=y
+CONFIG_SOC_PART_NUMBER_SAMD21G18A=y
+CONFIG_BOARD_ADAFRUIT_FEATHER_M0_LORA=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_GPIO=y
```

Comment:
Is `CONFIG_GPIO=y` needed for basic operation? If not, it should not be enabled on the board level.

Reply:
It can probably be removed. I based this on the equivalent in the Basic Proto defconfig ( https://github.com/zephyrproject-rtos/zephyr/blob/a61ca79041a0b8679a77a78228fff7698e2be1d2/boards/arm/adafruit_feather_m0_basic_proto/adafruit_feather_m0_basic_proto_defconfig#L10 ) .

If confirmed as not needed, should I file another PR for fixing the Basic Proto too ?

Reply:
Included in https://github.com/zephyrproject-rtos/zephyr/pull/51920